backend/test/invite_user_adm_handler_test.py

```diff
@@ -14,15 +14,24 @@
 
 from mock import patch
 
-def addAdminPermission(user, institution_key):
+def add_admin_permission(user, institution_key):
     user.add_permissions(permissions.DEFAULT_ADMIN_PERMISSIONS, institution_key)
 
-def hasAdminPermissions(user, institution_key):
+def add_super_user_permission(user, institution_key):
```

Comment:
This method does exactly the same thing as add_admin_permission, it would be better to pass the permission type to be added per parameter.